blackbox/docs/sql/statements/alter-cluster.rst

```diff
@@ -17,9 +17,11 @@ Synopsis
 
 ::
 
-    ALTER CLUSTER
-      { REROUTE RETRY FAILED }
-      | { SWAP TABLE source TO target [ WITH ( expr = expr [ , ... ] ) ] }
+    ALTER CLUSTER 
+      { REROUTE RETRY FAILED
+      | SWAP TABLE source TO target [ WITH ( expr = expr [ , ... ] ) ]
+      | GC DANGLING INDICES
```

Comment:
Should we call this `GC DANGLING ARTIFACTS` as we generally don't talk about indices and it would allow us to clean up non-indices in the future?

Reply:
cc: @joemoe

Reply:
I like `ARTIFACTS`

Reply:
yes, fine.